cmd/geninstaller/main.go

```diff
@@ -44,4 +47,12 @@ func main() {
 	defer w.Close() // nolint
 	err = installerTemplate.Execute(w, cli)
 	kctx.FatalIfErrorf(err)
+	if cli.PrintSHA256 {
+		err = w.Sync()
+		kctx.FatalIfErrorf(err)
+		script, err := os.ReadFile(cli.Dest)
+		kctx.FatalIfErrorf(err)
+		sha256sum := sha256.Sum256(script)
+		fmt.Printf("%v\n", hex.EncodeToString(sha256sum[:]))
```

Comment:
```suggestion
		fmt.Println(hex.EncodeToString(sha256sum[:]))
```

AIUI `fmt.Println` is implicitly `%v\n`